### DISCUSSION TITLE
Could I add more than one aircraft into a JSBSim instance?

### DISCUSSION THREAD

**Discussion by xiaodao2049** [JSBSim-Team/jsbsim#426] [Q&A]
Could I add more than one aircraft into a JSBSim instance? for example, 5 aircraft fly in a JSBSim environment.
If  JSBSim can add more than one aircraft into a instance , how can I do it ?

**Comment by seanmcleod** [comment]
You can simulate multiple aircraft within a single JSBSim process, however there will be a separate `FGFDMExec` instance managing each aircraft.

So for example, something along these lines:

```python
import jsbsim

aircraft1 = jsbsim.FGFDMExec('.', None)
aircraft1.load_script('scripts/c1721.xml')
aircraft1.run_ic()

aircraft2 = jsbsim.FGFDMExec('.', None)
aircraft2.load_script('scripts/737cruise.xml')
aircraft2.run_ic()

while True:
  aircraft1.run()
  aircraft2.run()
```

**Reply by xiaodao2049** [reply]
@seanmcleod Thanks a lot.
Can aircraft1 know the state of aircraft2? or aircraft2's traces can make an impact on aircraft1?
Actually, I want use the gym environment to wrap the JSBSim. however, I am not sure I can input the action_space of both two aircrafts into one step(actions) interface of gym.
